### DISCUSSION TITLE
Zen Cart 1.5.8b Documentation

### DISCUSSION THREAD

**Discussion by scottcwilson** [zencart/zencart#5862] [General]
I'm starting to write documentation about Zen Cart 1.5.8b.  You can view these changes on the documentation site. 

https://docs.zen-cart.com/user/upgrading/release_specific_upgrade_considerations/#zen-cart-158b

The What's New file entries are still commented out to avoid confusion.  You may view the What's New file entries by browsing the file directly: 

https://github.com/zencart/documentation/blob/98bf869adaf3443fdc67a73995c06f1c0feb62b2/content/release/whatsnew_1.5.8.html#L183

Please let me know if you made a change that you think should be listed here that isn't - it's quite difficult to build this list and it's never perfect the first time.

**Comment by brittainmark** [comment]
Am I being stupid! You say we have updated to Font awsome 6 yet all the files are still 4.7.  Is this still to be added to the repository?

**Comment by scottcwilson** [comment]
This will be true as soon as #5845 is merged.  Remember that anything written about 1.5.8b will include things that are on the come.

**Comment by brittainmark** [comment]
Thanks for the clarification. I was getting ready to test my site with fontawsome 6.

**Comment by scottcwilson** [comment]
Closing this thread since we are renumbering the next Zen Cart Release. 
See https://docs.zen-cart.com/user/about_us/versioning/
